tests/foreman/cli/test_activationkey.py

```diff
@@ -951,25 +951,8 @@ def test_info_activation_keys_1(self):
 
         """
 
-    @stubbed()
-    def test_info_activation_keys_2(self):
-        """@Test: Get Activation key info for all variations of Description
-
-        @Feature: Activation key - info
-
-        @Steps:
-
-        1. Create Activation key for all valid Description variation in [1]
-        2. Get info of the Activation key
-
-        @Assert: Activation key info is displayed
-
-        @Status: Manual
-
-        """
-
     @tier1
-    def test_bugzilla_1111723(self):
+    def test_positive_create_using_old_name(self):
```

Comment:
This test scenario is little different so `test_positive_create_with_same_name` may not work.
* Create an activation key
* Rename it
* Create another activation key with the same name from step 1